src/components/Landingpage.js

```diff
@@ -0,0 +1,14 @@
+
+function Landingpage() {
+  return (
+    <div>
+      <div className="btns">
+        <button className="btn">Sign Up</button>
+        <h4>I have an account</h4>
+        <button className="btn">Sign In</button>
+      </div>
+    </div>
+  );
+}
+export default Landingpage;
+
```

Comment:
It would be nice if the `LandingPage` component is in it's own folder along with it's own CSS file. It would  help if you ever wanna make modifications to the styles, that way you wouldn't have to go through too many lines of code to make the change.